lib/KDropdownMenu.vue

```diff
@@ -162,8 +165,31 @@
         ) {
           this.focusOnButton();
         }
+
+        if (this.lastFocusElement) {
```

Comment:
Above this, you can see there is a call to the `this.focusOnButton()` method, which apparently was an old behaviour we had in KDropdownMenu, but its no longer useful as now KDropdownMenu doesnt render any button anymore.

But if in the past, we were making this call in those conditions, it was for some reason (perhaps because of an edge case im not aware of), but to play safe, lets just replace that `focusOnButton()` call with this lastFocusElement call, for this focus to occur on those conditions too, so we would have something like:
```javascript
        if (
          popover.contains(focusedElement) &&
          (focusedElement.classList.contains('ui-popover') ||
            focusedElement.classList.contains('ui-popover-focus-redirector') ||
            focusedElement.classList.contains('ui-menu-option'))
        ) {
          if (this.lastFocusElement) {
            this.lastFocusElement.focus();
          }
        }
```
And then we can remove the `focusOnButton` method :).